.travis.yml

```diff
@@ -43,15 +43,11 @@ matrix:
   include:
     - rust: stable
       env: FEATURES='default'
-    - rust: stable
-      env: FEATURES='alloc'
     - rust: stable
       env: FEATURES=''
 
     - rust: beta
       env: FEATURES='default'
-    - rust: beta
-      env: FEATURES='alloc'
```

Comment:
don't remove these; the jobs should be in both (yeah I know, it doesn't make sense, but that's how Travis works)

Reply:
Done